File: src/main/java/ejercicio13/Main.java
package ejercicio13;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner lectura = new Scanner (System.in);
        System.out.print("Dime el nombre del anime: ");
        String nombre = lectura.nextLine();

        System.out.print("Dime los capitulos del anime: ");
        int episosdios = lectura.nextInt();

        lectura.nextLine();

        System.out.print("Dime el genero del anime: ");
        String genero = lectura.nextLine();

        Anime animeCreated = new Anime(nombre,episosdios,genero);
        animeCreated.mostrarInfo();

    }
}

File: src/main/java/ejercicio13/Main.kt
package ejercicio13

class Anime2(private val nombre: String, private val episodios: Int, private val genero: String) {
    fun mostrarInfo() {
        println("Nombre: $nombre, Episodios: $episodios, Género: $genero")
    }
}

fun main() {
    fun main() {
        print("Dime el nombre del anime: ")
        val nombre = readln()

        print("Dime los capítulos del anime: ")
        val episodios = readln().toInt()

        print("Dime el género del anime: ")
        val genero = readln()

        val animeCreated = Anime2(nombre, episodios, genero)
        animeCreated.mostrarInfo()
    }
}